pkg/cmd/extension/http.go

```diff
@@ -19,26 +19,25 @@ func hasScript(httpClient *http.Client, repo ghrepo.Interface) (hs bool, err err
 	url := ghinstance.RESTPrefix(repo.RepoHost()) + path
 	req, err := http.NewRequest("GET", url, nil)
 	if err != nil {
-		return
+		return false, err
```

Comment:
Looks like we have explicit return parameters now, can we remove the named return parameters to reduce confusion going forward?